classes/CartRule.php

```diff
@@ -455,7 +455,7 @@ public static function getCustomerCartRules(
             if ($cart_rule['shop_restriction']) {
                 $cartRuleShops = Db::getInstance()->executeS('SELECT id_shop FROM ' . _DB_PREFIX_ . 'cart_rule_shop WHERE id_cart_rule = ' . (int) $cart_rule['id_cart_rule']);
                 foreach ($cartRuleShops as $cartRuleShop) {
-                    if (Shop::isFeatureActive() && ($cartRuleShop['id_shop'] == Context::getContext()->shop->id)) {
+                    if ($cartRuleShop['id_shop'] == Context::getContext()->shop->id) {
```

Comment:
Be careful this is a huge change here, I believe the shop is always present but it doesn't mean the Multi shop is really active (Multishop settings `PS_MULTISHOP_FEATURE_ACTIVE` enabled + more than one shop created).

If the code previously checked this condition about the feature being active maybe it was for good reasons, why do you need to change this?